Build/MixedReality.Toolkit/MixedReality.Toolkit.csproj

```diff
@@ -0,0 +1,26 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <EnableDefaultCompileItems>false</EnableDefaultCompileItems>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Compile Include="$(MRTKRoot)\Assets\MixedRealityToolkit\**\*.cs"  />
+
+    <Content Include="$(MRTKRoot)\Assets\MixedRealityToolkit\_Core\Resources\**\*">
```

Comment:
Potentially. I care a little less about where they are put, and more that they exist and work as intended. I haven't looked at the code that looks for these dynamically, but I know I hit an issue if I changed the structure too much. This might be worth a follow up issue to track

Reply:
True, definitely a nit. =)